### PR TITLE
Add create function for AlphaNumStringType argument

### PR DESCRIPTION
**Pull request by InformatiqueProg**

### CODE REVIEW

**Comment by gillesbourgeat** [comment]
Hello,
The method createAnyTypeArgument does not suit you ?

**Comment by InformatiqueProg** [comment]
Why not.
I need an argument with only "alphaNum" and there is a thelia Type for this, that's why...

File: core/lib/Thelia/Core/Template/Loop/Argument/Argument.php [hunk 1]
```diff
@@ -176,4 +175,46 @@ public static function createEnumListTypeArgument($name, array $entries, $defaul
...
+
+    /**
+     * @param $name
+     * @param null $default
+     * @param bool $mandatory
+     * @param bool $empty
+     * @return Argument
+     * @since 2.3.0
+     */
+    public static function createAlphaNumStringListType($name, $default = null, $mandatory = false, $empty = true)
```
Comment (gillesbourgeat) [inline core/lib/Thelia/Core/Template/Loop/Argument/Argument.php]: Ok,
You forgot the suffix "Argument" for the name of this method.

**Comment by gillesbourgeat** [comment]
Hello,
Can you change the "@since" documentation to version 2.4.0 and squash your commits.
Thanks !

**Comment by InformatiqueProg** [comment]
Done

**Comment by gillesbourgeat** [comment]
Thanks !

**Pull request merged**